src/GeekPlatform/Views/Profil/Index.cshtml

```diff
@@ -15,14 +18,14 @@
         <div class="panel panel-profile">
             <div class="panel-heading">Általános adatok</div>
             <div class="panel-body">
-                <h3 style="margin-top: 0px;">Név</h3>
-                <h4><em>pozíció, team tagság</em></h4>
+                <h3 style="margin-top: 0px;">@Model.Nev</h3>
+                <h4><em>@Model.PozicioTeamTagsag</em></h4>
                 <hr>
                 <div class="row">
                     <div class="col-xs-4">
-                        <p>Tagság kezdete:</p>
-                        <p>Tagság várható vége:</p>
-                        <p>Munkahely, pozíció:</p>
+                        <p>@Model.TagsagKezdete:</p>
```

Comment:
Ezeknél a felsorolálsos adatoknál (`Tagság kezdete`, `Tagság várható vége`, `Munkahely, pozíció`) marajdon meg a kíírt tulajdonság is, és mellé tegyük be az értéket, ahogy a [képen](https://drive.google.com/file/d/0Bz62sjhk4NCfRU1BR3R0ZXJuVTQ/view)

Tehát az itt lejjebbi `<p>Teszt szöveg, ami elég hosszú</p>` részek helyére jöhetnének a modelles adatok